clonelists/Commodore - Amiga CD32 (Redump).json

```diff
@@ -14,6 +18,14 @@
 			"compilations": [
 				{"searchTerm": "Grandslam Gamer Gold Collection"}
 			]
+		},		
+		{
```

Comment:
You should also add the "Wing Commander" group for the compilation, even though there isn't an individual title. This helps with compilation handling.

```json
{
  "group": "Wing Commander",
  "compilations": [
    {"searchTerm": "Dangerous Streets & Wing Commander"}
  ]
}
```